src/services/itemMembership/index.ts

```diff
@@ -36,16 +34,12 @@ const plugin: FastifyPluginAsync = async (fastify) => {
         fastify.register(fastifyCors, fastify.corsPluginOptions);
       }
 
-      // if (WEBSOCKETS_PLUGIN) {
-      //   registerItemMembershipWsHooks(
-      //     websockets,
-      //     runner,
-      //     itemsDbService,
-      //     dbService,
-      //     taskManager,
-      //     db.pool,
-      //   );
-      // }
+      registerItemMembershipWsHooks(
+        buildRepositories(),
+        websockets,
+        itemService,
+        itemMembershipService,
+      );
```

Comment:
Looks like a review comment I was adding got removed by the cat walking on the keyboard 🐈 :

In the items I have packed the destructuration and the call into a fastify plugin, so that it's more consistent:

https://github.com/graasp/graasp/blob/36223a3508df54ce6ff5a75c689449ed02b547da/src/services/item/ws/hooks.ts#L180-L184

So the registration now looks like this:

https://github.com/graasp/graasp/blob/36223a3508df54ce6ff5a75c689449ed02b547da/src/services/item/index.ts#L101

We may want to do the same here